sections/rust.zsh

```diff
@@ -11,7 +11,7 @@
 SPACESHIP_RUST_SHOW="${SPACESHIP_RUST_SHOW=true}"
 SPACESHIP_RUST_PREFIX="${SPACESHIP_RUST_PREFIX="$SPACESHIP_PROMPT_DEFAULT_PREFIX"}"
 SPACESHIP_RUST_SUFFIX="${SPACESHIP_RUST_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
-SPACESHIP_RUST_SYMBOL="${SPACESHIP_RUST_SYMBOL="𝗥 "}"
+SPACESHIP_RUST_SYMBOL="${SPACESHIP_RUST_SYMBOL="🦀 "}"
```

Comment:
Quick note, With this symbol color won't be affected with `SPACESHIP_RUST_COLOR`. So does some other language sections.

Reply:
Hello!
So which file do I need to update for the ferris?